frontend/src/components/NewPassword/NewPassword.js

```diff
@@ -22,14 +27,43 @@ class NewPassword extends Component {
     this.setState({ [event.target.name]: event.target.value });
   };
 
+  handleSubmit = (event, changeCreatedState) => {
+    event.preventDefault();
+    const { history } = this.props;
+    const { password, confirmPassword } = this.state;
+    const { resetPasswordToken } = this.state;
+    const queryString = `resetpassword/?q=${resetPasswordToken}`;
+    if (password !== confirmPassword) {
+      alert("Passwords don't match"); // eslint-disable-line no-undef
+    } else {
+      resetPassword(this.state)
+        .then((response) => {
+          changeCreatedState(response.status);
+          history.push(queryString);
```

Comment:
I don't think we want to be redirecting from here to the reset password page, as that's where we are. We could just redirect to the homepage?